cypress/integration/brewery/ScenarioLaunch-test.spec.js

```diff
@@ -15,6 +15,10 @@ describe('Create scenario', () => {
     scenariosNames[i] = SCENARIO_NAME.SCENARIO_MASTER + i + ' - ' + randomString;
   }
 
+  Cypress.Keyboard.defaults({
+    keystrokeDelay: 0,
+  });
+
```

Comment:
We may need this change of keystrokeDelay value for the whole test file, can we move these lines at the top of the file, right after the imports ? (like we did in test files for Table and File parameters)